cypress/e2e/middleware/enhanced.cy.ts

```diff
@@ -13,6 +13,12 @@ describe('Enhanced middleware', () => {
     })
   })
 
+  it.skip('passes in headers within request.rewrite()', () => {
```

Comment:
Should this be skipped?

Reply:
ah this one should not be skipped, thanks!